coding_introduction/CodingIntroduction.md

```diff
@@ -308,6 +373,33 @@ Regular files have filemode 644.
 - use `git add *`, **NEVER**.
 
 
+### Best Practices: Pull Requests
+For Pull Requests, there are two kind of best practices, related to the Pull Request itself, and to the Pull Request
+Review, that we established in [@ipa320](https://github.com/ipa320).
+
+- Pull Request
+  - keep the PR small (<100 changed files, <1000 changed lines of code)
```

Comment:
<100 Files and <1000 LOC are the max upper bounds. I'd rather like to specify <10 Files and <100 LOC